### PR TITLE
runtime/syntax/vim.vim: fixes for vimAugroupList

### PR DESCRIPTION
**Pull request by blueyed**
Similar issue, for when merging / picking this: https://github.com/vim/vim/issues/2450

### CODE REVIEW

**Comment by blueyed** [comment]
Started based on this issue:

The second comment in the following code is not highlighted as a comment, but `vimFilter vimAugroup`:

```vim
augroup foo
  if 1  " comment
  endif
  if !1  " comment
  endif
augroup END
```

File: runtime/syntax/vim.vim [hunk 1]
```diff
@@ -166,7 +166,7 @@ syn match vimNumber	"-\d\+\%(\.\d\+\%([eE][+-]\=\d\+\)\=\)\="  skipwhite nextgro
 syn match vimNumber	"\<0[xX]\x\+"		       skipwhite nextgroup=vimGlobal,vimSubst,vimCommand,vimComment
 syn match vimNumber	"\%(^\|\A\)\zs#\x\{6}"             	       skipwhite nextgroup=vimGlobal,vimSubst,vimCommand,vimComment
 
-" All vimCommands are contained by vimIsCommands. {{{2
+" All vimCommands are contained by vimIsCommand. {{{2
```
Comment (chdiza) [inline runtime/syntax/vim.vim:169]: Changes to this file are supposed to be emailed to Dr. Chip.

Comment (blueyed) [reply runtime/syntax/vim.vim:169]: I assume that is @cecamp then?

I think it is better/good to have patches here, which allowed me to find e.g. https://github.com/vim/vim/issues/2450 in the first place.

Comment (chdiza) [reply runtime/syntax/vim.vim:169]: > I think it is better/good to have patches here

That is not The Vim Way.

Comment (blueyed) [reply runtime/syntax/vim.vim:169]: I've emailed Dr. Chip.

**Pull request closed without merging**